Assets/Scripts/Shootable/WanderAI.cs

```diff
@@ -8,22 +8,40 @@ public class WanderAI : MonoBehaviour
     private MeshRenderer meshRenderer;
     private Canvas alertCanvas;
     private float perceptionRadius;
+    private float sneakingPerceptionRadius;
+    private float height;
+    private int fieldOfView;
```

Comment:
As seen in the spec, some furniture may have multiple vision cones, and for some, the vision cones are spherical instead. I would personally suggest creating a `Sense` scriptable object and creating `SenseCone` and SenseSphere` scriptable objects that can determine if a sound or the player is in them, and which can be dragged onto a piece of furniture, but you are welcome to explore other avenues.